lntest/itest/lnd_test.go

```diff
@@ -12717,12 +12717,16 @@ func testAbandonChannel(net *lntest.NetworkHarness, t *harnessTest) {
 	}
 
 	// Make sure the channel is no longer in the channel backup list.
-	bkupAfter, err := ioutil.ReadFile(net.Alice.ChanBackupPath())
+	err = wait.Predicate(func() bool {
```

Comment:
If I understand this correctly, this is going to read Alice's channel backup every 20ms until the backup grows, unless timeout passed?

Reply:
Yes, that's correct.